docs/walkthrough_collaborate.md

```diff
@@ -2,19 +2,72 @@
 title: Collaborate within eScriptorium
 summary: About features related to collaboration in eScriptorium.
 authors:
-    - Alix Chagué
-date: 2023-01-31
+    - Hugo Scheithauer, Alix Chagué
+date: 2023-02-09
 ---
 
-# Walkthrough : collaborate within eScriptorium
+# Walkthrough: collaborate within eScriptorium
 
-!!! warning
-    The section needs to be completed.
+This walkthrough details features related to collaboration in eScriptorium.
 
-## Create a team or join a team
+## Teams
+
+Users can be grouped by Teams. The main purpose of a team is to facilitate [sharing documents or projects](#share-a-project-or-a-document). Teams are also helpful to identify a group of people working on the same project or coming from the same institution.  
+
+It is possible to manage teams from the [Profile page](walkthrough_users.md). Team management includes:  
+
+- Creating a team,  
+- Leaving a team,  
+- Adding or removing members from a team,  
+- Transfering ownership to another member.  
+
+Only the owner of a team is allowed to add or remove members or to transfer ownership. The owner of a team cannot leave it without first transferring the ownership to another user.
+
+??? Note "No deleting, no renaming"
+    It is currently impossible to delete a team or to rename a team.
+
+??? Note "One owner per team"
+    There can only be one owner at a time.
 
 ## Invite a user
 
-## Share a project or a document 
+eScriptorium allows you to invite new users to your instance.
```

Comment:
allows **some users** to invite. The invitation rights are not activated for everyone